trac-env/htdocs/tickethacks.js

```diff
@@ -207,4 +207,110 @@ $(function() {
         }
         $("table.properties").append("<tr><th>Pull Requests:</th><td>" + link + "</td><tr>");
     });
+
+    // Ticket Triage Guidelines: show next steps based on the ticket's status,
+    // flags, etc.
+    function get_boolean_ticket_flag(name) {
+        return $('#h_' + name).next().text().trim() === 'yes';
+    }
+    function get_ticket_flag(name) {
+        return $('#h_' + name).next().text().trim();
+    }
+    var stage = get_ticket_flag('stage');
+    var ticket_status = $('.trac-status').text().trim();
+    var ticket_type = $('.trac-type').text().trim();
+    var has_patch = get_boolean_ticket_flag('has_patch');
+    var patch_needs_improvement = get_boolean_ticket_flag('needs_better_patch');
+    var needs_docs = get_boolean_ticket_flag('needs_docs');
+    var needs_tests = get_boolean_ticket_flag('needs_tests');
+    var next_steps = [];
+    var include_link_to_pr_msg = (
+        "include a link to the pull request in the ticket comment when making " +
+        "that update. The usual format is: [https://github.com/django/django/pull/####&nbsp;PR]."
+    );
+    if (ticket_status == 'closed') {
+        // TODO (e.g. reopening a wontfix or needsinfo, or what to do in the
+        // case of a regression caused by the ticket
+    } else if (stage == 'Unreviewed') {
+        next_steps.push(
+            "For bugs: reproduce the bug. If it's a regression, " +
+            "<a href='https://docs.djangoproject.com/en/dev/internals/contributing/triaging-tickets/#bisecting-a-regression'>" +
+            "bisect</a> to find the commit where the behavior changed.<br>"
+        );
+        next_steps.push(
+            "For new features or cleanups: give a second opinion of the proposal. " +
+            "Complex proposals may need to be discussed on the " +
```

Comment:
I think the DEP requirement isn't common enough to mention it here. Actually the link to django-developers might be too much too. The triager could mention that requirement if they feel it's needed.